data/data/powervs/variables-powervs.tf

```diff
@@ -48,7 +48,7 @@ variable "powervs_service_instance_name" {
 variable "powervs_wait_for_workspace" {
   type        = string
   description = "The seconds wait for the Power VS Workspace after creation, default is 60s"
```

Comment:
It should say `3m` now, but I wonder if it needs to say anything...

Reply:
yeaaah Rafael pointed that out as well. If we do any further terraform updates we can update this